scripts/ci-build-kubernetes.sh

```diff
@@ -73,8 +73,8 @@ setup() {
 
     # Docker tags cannot contain '+'
     # ref: https://github.com/kubernetes/kubernetes/blob/5491484aa91fd09a01a68042e7674bc24d42687a/build/lib/release.sh#L345-L346
-    export IMAGE_TAG="${KUBE_GIT_VERSION/+/_}"
-    echo "using K8s IMAGE_TAG=${IMAGE_TAG}"
+    export KUBE_IMAGE_TAG="${KUBE_GIT_VERSION/+/_}"
```

Comment:
I ended up not adding a back-compat line for the variable name change since it's exported here anyways and whatever the variable would have been set to is ignored